Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.9.6-slim-buster as parent
+
+RUN apt-get update && apt-get install -y curl && \
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python && \
+    echo 'export PATH="$HOME/.poetry/bin:$PATH"' >> $HOME/.profile && \
+    /bin/bash -c "source $HOME/.profile"
+
+ENV PATH="${HOME}/.poetry/bin:${PATH}"
+
+WORKDIR /app
+
+COPY . /app
+
+RUN  $HOME/.poetry/bin/poetry install --no-dev --no-root
```

Comment:
Worth noting that this means changing any copied file will force a re-install because we lose the caching. This is mostly fine for our use case (particularly if we're using the bind-mount for dev) but we might still find it valuable to split this into e.g.
```docker
COPY poetry.lock pyproject.toml /app/
RUN $HOME/.poetry/bin/poetry install --no-dev --no-root
COPY . /app
```

So that changing _most_ files doesn't invalidate the layer cache

Reply:
Thanks, done